benchmarks/src/main/java/org/elasticsearch/benchmark/compute/operator/AggregatorBenchmark.java

```diff
@@ -575,13 +575,14 @@ private static void run(String grouping, String op, String blockType, int opCoun
         };
 
         DriverContext driverContext = driverContext();
-        Operator operator = operator(driverContext, grouping, op, dataType);
-        Page page = page(driverContext.blockFactory(), grouping, blockType);
-        for (int i = 0; i < opCount; i++) {
-            operator.addInput(page);
+        try (Operator operator = operator(driverContext, grouping, op, dataType)) {
+            Page page = page(driverContext.blockFactory(), grouping, blockType);
+            for (int i = 0; i < opCount; i++) {
+                operator.addInput(page.shallowCopy());
```

Comment:
do we need to call `shallowCopy`?

Reply:
Yeah, we build the page one time outside the loop. Without that it crashes on the second run.